httpx/client.py

```diff
@@ -57,6 +58,7 @@ def __init__(
         cookies: CookieTypes = None,
         verify: VerifyTypes = True,
         cert: CertTypes = None,
+        http_versions: HTTPVersionTypes = None,
```

Comment:
Should we have an explicit DEFAULT_... here instead of None?

Reply:
For the `None` cases I don't think so, no. We're being consistent with the surrounding context.